docs/operator-manual/argocd-cm.yaml

```diff
@@ -101,7 +101,7 @@ data:
       ignoreDifferences: |
         jsonPointers:
         - webhooks/0/clientConfig/caBundle
-    certmanager.k8s.io/Certificate:
```

Comment:
don't we need both of these?

Reply:
As a document, I thought it is enough to use only the latest cert-manager API.
Is it better to write some comments here or add `resource.customizations` 
for old and new cert-manager API? 🤔 

(This pull request supports the old and new cert-manager API.)

Reply:
I don’t think we want to break backwards compatibility.

Reply:
Thanks! I understood. 
I will fix this immediately.